plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -194,7 +194,9 @@ const CatalogPageContents = (props: CatalogPageProps) => {
               Add example components
             </Button>
           )}
-          <SupportButton>All your software catalog entities</SupportButton>
+          <SupportButton title="Nobody">
```

Comment:
This doesn't seem to informative? 😅 Should just move the existing child into the `title` right?

Reply:
@Rugvip I forgot to move it😅